enforcer_test.go

```diff
@@ -198,6 +198,41 @@ func TestNotUsedRBACModelInMemory(t *testing.T) {
 	testEnforce(t, e, "bob", "data2", "write", true)
 }
 
+func TestMatcherUsingInOperator(t *testing.T) {
+	// In memory
```

Comment:
Is it necessary to load from model file and text string at the same time? Both ways are already well tested in other test cases. And this is not this PR's focus. I think just using model file only will be OK.

Reply:
That's right, I just want to prove that it will also work well in in-memory mode. 😁